### PR TITLE
Obfuscation on folder names

### PR DESCRIPTION
**Pull request by tjungblu**

### CODE REVIEW

File: pkg/traversal/worker.go [hunk 1]
```diff
@@ -107,22 +106,22 @@ func (w *worker) run() {
...
 			}
 		}
 
-		// obfuscate the name if required
-		newName := wf.f.Name()
+		originalPath := wf.f.Path()
+		newPath := originalPath
 		for _, o := range w.obfuscators {
-			newName = o.FileName(newName)
+			newPath = o.Path(newPath)
```
[the remark this answers is not in the document]
Comment (tjungblu) [reply pkg/traversal/worker.go:117]: @arjunrn but do you want the FileWalker to do obfuscation as well?

[the remark this answers is not in the document]
Comment (arjunrn) [reply pkg/traversal/worker.go:117]: I don't think that's an issue.

[the remark this answers is not in the document]
Comment (arjunrn) [reply pkg/traversal/worker.go:117]: Or on second thoughts we don't want that.

**Pull request merged**